src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -16,6 +17,14 @@
           Condition="'$(RestorePackages)'=='true' AND '$(IsTestProject)' == 'true' AND '$(TestRuntimeProjectJson)' != ''">
     <Exec Command="$(DnuRestoreCommand) &quot;$(TestRuntimeProjectJson)&quot;" StandardOutputImportance="Low" CustomErrorRegularExpression="^Unable to locate .*" />
   </Target>
+  
+  <Target Name="ValidateTestRuntimeDependencies"
+          AfterTargets="RestoreTestRuntimePackage"
```

Comment:
@ericstj @weshaggard When I enable this and build from the corefx repo root, the same set of validation errors are reported for each project (if there are any errors). Is there a better way to hook this target in, or is that behavior ok?